test/Orleans.CodeGenerator.Tests/snapshots/OrleansSourceGeneratorTests.TestBasicClass.verified.cs

```diff
@@ -0,0 +1,121 @@
+﻿#pragma warning disable CS1591, RS0016, RS0041
```

Comment:
these files could be useful down the road for explaining what the source-generator does :)